tests/clone_test.go

```diff
@@ -698,8 +700,24 @@ var _ = Describe("[Serial]VirtualMachineClone Tests", Serial, func() {
 						libinstancetype.WaitForVMInstanceTypeRevisionNames(sourceVM.Name, virtClient)
 
 						if toRunSourceVM {
-							By("Starting the VM and expecting it to run")
-							sourceVM = RunVMAndExpectLaunchWithRunStrategy(virtClient, sourceVM, virtv1.RunStrategyAlways)
+							runStrategy := virtv1.RunStrategyAlways
+							sourceVM.Spec.RunStrategy = &runStrategy
+							patchSet := patch.New()
+							patchSet.AddOption(
+								patch.WithAdd("/spec/runStrategy", virtv1.RunStrategyAlways),
+								patch.WithAdd("/spec/running", nil),
+							)
+							payload, err := patchSet.GeneratePayload()
+							Expect(err).ToNot(HaveOccurred())
+							sourceVM, err = virtClient.VirtualMachine(sourceVM.Namespace).Patch(context.Background(), sourceVM.Name, types.JSONPatchType, payload, v1.PatchOptions{})
+							Expect(err).ToNot(HaveOccurred())
+
+							By("VM has the running condition")
+							Eventually(func() bool {
+								vm, err := virtClient.VirtualMachine(sourceVM.Namespace).Get(context.Background(), sourceVM.Name, v1.GetOptions{})
+								Expect(err).ToNot(HaveOccurred())
+								return vm.Status.Ready
+							}, 300*time.Second, 1*time.Second).Should(BeTrue())
```

Comment:
Please use `WithTimeout` and `WithPolling` instead of anonymous interval parameters.

Reply:
```suggestion
							}).WithTimeout(300*time.Second).WithPolling(1*time.Second).Should(BeTrue())
```

Reply:
Done :)